source/chapters/library.rst

```diff
@@ -313,10 +314,12 @@ Tracks can be loaded into a deck in several ways:
   load a track into a deck.
 * By :ref:`control-keyboard` to load the selected track in the library track
   table.
+* Using keyboard :term:`shortcut <shortcuts>` keys to load track in Deck 1 with :kbd:`Shift` + :kbd:`Left` and Deck 2
+  with :kbd:`Shift` + :kbd:`Right`
 * Drag-and-drop from library track table: Dragging-and-dropping a track from the
   track table onto a waveform display will load the track into that deck.
 * Drag-and-drop from deck to deck: Once you've loaded a track to deck, sampler,
-  or preview deck, click on the :ref:`track title <interface-track-info>` and
+  or preview deck, click on the :ref:`track title <interface-track-info>`, :ref:`track artist <interface-track-info>` or cover art and
```

Comment:
please also mention Playlists / Crates here.

Reply:
Not clear on this

Reply:
the other way around: you can drag tracks to playlists and crates.
Do you have Mixxx up and running and can test it yourself to get a feeling for the UX?

Reply:
Yes, I have it installed. I knew that the properties can be dragged to playlists and crates, I just thought the section was concentrating on just decks since it starts with "Drag -and-drop from deck to deck"

Reply:
okay, nice.
You're right, this may not be the best spot to mention deck-to-playlist dragging, but right now there's no chapter where it would fit better without redundancy IMO.
How about adding this to the same bullet point:
```
Note you can also drop tracks onto Playlists and Crates in the sidebar, as well as into track tables of active Playlists or Crates.
```

Reply:
Yeah, I think that will be better. I'll do that shortly. Thank you